content/md/franklin-insights/franklin-insights-mappings.md

```diff
@@ -59,14 +59,20 @@ _Since the 3.1 version_, unmapped attributes are displayed first.
 
 ### Exact match
 
-No need to map attributes if they're the same, we do it for you: the PIM attribute is automatically mapped and saved, the proposal is automatically generated!
+No need to map attributes if they're the same, we do it for you: the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```

Comment:
```suggestion
No need to map attributes if they are the same, we do it for you: the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```
I'd rather use the full word instead of a contraction